docs/content/AnchoredOverlay.mdx

```diff
@@ -35,3 +37,121 @@ See also [Overlay positioning](/Overlay#positioning).
   }}
 </State>
 ```
+
+## Props
+
+<PropsTable>
+  <PropsTableRow
+    name="open"
+    type="boolean"
+    defaultValue="false"
+    description="Determines whether the overlay portion of the component should be shown or not."
+  />
+  <PropsTableRow
+    name="onOpen"
+    type="(gesture: 'anchor-click' | 'anchor-key-press') => unknown"
+    description='A callback that is called whenever the overlay is currently closed and an "open gesture" is detected.'
+  />
+  <PropsTableRow
+    name="onClose"
+    type="(gesture: 'anchor-click' | 'click-outside' | 'escape') => unknown"
+    description='A callback that is called whenever the overlay is currently open and a "close gesture" is detected.'
+  />
+  <PropsTableRow
+    name="renderAnchor"
+    type="<T extends React.HTMLAttributes<HTMLElement>>(props: T) => JSX.Element"
+    description={
+      <>
+        A custom function component used to render the anchor element. When renderAnchor is null, an anchorRef is
+        required.
+      </>
+    }
+  />
+  <PropsTableRow
+    name="anchorRef"
+    type="React.RefObject<HTMLElement>"
+    description={
+      <>
+        An override to the internal <InlineCode>renderAnchor</InlineCode> ref that will be used to position the overlay.
+        When <InlineCode>renderAnchor</InlineCode> is
+        <InlineCode>null</InlineCode>, this can be used to make an anchor that is detached from <InlineCode>
```

Comment:
This wording is tricky — unlike most ref props, there's an expectation that the `AnchoredOverlay` isn't responsible for setting the value of the ref, right? only that it will use the ref's current value for positioning logic?

Reply:
Yeah, this one is tricky. I mostly copied this description from the code comments but I agree that it's a bit confusing.